### PR TITLE
Move partition labels to distro options

### PR DESCRIPTION
**Pull request by ToKe79**
I propose this change to have easier control over the partition labels. If there will be any need to change the partition labels in future, these changes can be done then in one place. Also useful for projects based on LibreELEC, which use different partition labels (the original reason for this change - keep the upstream code as is and if anything needs to be changed, try to make the change work for everybody).

Thank you for your consideration.
-vudiq

### CODE REVIEW

**Comment by MilhouseVH** [comment]
Thanks - looks fine to me. Any Amlogic guys with any concerns/thoughts?

**Comment by chewitt** [comment]
I get OCD issues when THERE_ARE_TOO_MANY_UNDERSCORES in variables. Is there any good reason we cannot just use `BOOT_LABEL` and `DISK_LABEL` everywhere? Use the #comment above them to add more context if necessary, e.g. `# Installation USB/SD disk partition labels`

**Comment by ToKe79** [comment]
`DISK_LABEL` is already used in `scripts/mkimage`:
https://github.com/LibreELEC/LibreELEC.tv/blob/66a913dac572273ee51490db39b052d51b58eca7/scripts/mkimage#L36-L40
The `DISTRO_` should point to the fact that the variable is defined on the distribution level (`distributions/*/options`). So I changed the new variables to `DISTRO_BOOTLABEL` and `DISTRO_DISKLABEL` respectively to have less underscores.

**Comment by ToKe79** [comment]
I just noticed the changes of #2721 - I found one place where hard-coded partition labels were removed and some other where they were added:
`projects/Amlogic/initramfs/platform_init` (modified file)
`projects/Amlogic/packages/autoscript-amlogic/instboot/data_to_emmc_uEnv.ini` (new file)
`projects/Amlogic/packages/autoscript-amlogic/instboot/uEnv.ini` (new file)

File: scripts/mkimage [hunk 1]
```diff
@@ -71,8 +71,8 @@ trap cleanup SIGINT
   UUID_2=$(date '+%M%S')
   FAT_SERIAL_NUMBER="${UUID_1}${UUID_2}"
   UUID_SYSTEM="${UUID_1}-${UUID_2}"
-  FAT_VOLUME_LABEL="LIBREELEC"
-  STORAGE_VOLUME_LABEL="STORAGE"
+  FAT_VOLUME_LABEL="${DISTRO_BOOTLABEL}"
```
Comment (chewitt) [inline scripts/mkimage]: You can simplify mkimage by changing instances of `$FAT_VOLUME_LABLE` to be `$DISTRO_BOOTLABEL` and `$STORAGE_VOLUME_LABEL` to `$DISTRO_DISKLABEL`

**Pull request merged**